package.json

```diff
@@ -88,6 +88,11 @@
           "enum": [
             "Truffle"
           ]
+        },
+        "truffle-vscode.enableTelemetry": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Indicates whether to collect anonymous product usage data. Regardless of this setting, we honor the [`telemetry.telemetryLevel`](https://code.visualstudio.com/docs/getstarted/telemetry#_disable-telemetry-reporting) user setting, _i.e._, if `telemetry.telemetryLevel` is `off` we do not send any data regardless of this setting. See <https://trufflesuite.com/analytics/> for more info"
```

Comment:
@kevinbluer you might want to improve this `description`. Is there any other telemetry policy to mention here?